posthog/hogql/errors.py

```diff
@@ -46,7 +46,7 @@ class InternalHogQLError(BaseHogQLError):
 # Specific exceptions
 
 
-class SyntaxError(ExposedHogQLError):
+class SyntaxHogQLError(ExposedHogQLError):
```

Comment:
I think it would look more natural as "HogQLSyntaxError". Maybe a convention like this:
- "{Base,Exposed,Internal}HogQLError" – the base classes you shouldn't use directly.
- "HogQL{Foo}Error" – concrete, usable error types